lib/fizzy/utf8.cpp

```diff
@@ -105,11 +105,12 @@ bool utf8_validate(const uint8_t* pos, const uint8_t* end) noexcept
         else
             return false;
 
+        assert(required_bytes > 1);
```

Comment:
Could put this line after the comment below

Reply:
Oh yeah, good point.